File: app/src/main/java/de/kasoki/trierbustimetracker/DonateActivity.java
package de.kasoki.trierbustimetracker;

import android.app.Activity;
import android.graphics.Color;
import android.graphics.drawable.ColorDrawable;
import android.os.Bundle;
import android.view.MenuItem;
import android.webkit.WebView;

import de.kasoki.trierbustimetracker.utils.Helper;

public class DonateActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_donate);
        setupTheme();

        if(Helper.getCurrentAPILevel() >= 7) {
            this.addHomeAsUpButtonToActionBar();
        }

        WebView web = (WebView) this.findViewById(R.id.webview);
        web.loadUrl(this.getString(R.string.donate_url));
    }

    private void setupTheme() {
        getActionBar().setBackgroundDrawable(
                new ColorDrawable(Color.parseColor(MainActivity.ACTIONBAR_COLOR)));
    }

    private void addHomeAsUpButtonToActionBar() {
        getActionBar().setDisplayHomeAsUpEnabled(true);
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {

        switch (item.getItemId()) {
            case android.R.id.home:
                finish();
                return true;
            default:
                return super.onOptionsItemSelected(item);
        }
    }
}

File: src/main/scala/de/kasoki/trierbustimetracker/DonateActivity.scala
package de.kasoki.trierbustimetracker

import org.scaloid.common._
import android.os.Bundle
import android.view.Menu
import android.view.MenuItem
import android.webkit.WebView

import de.kasoki.trierbustimetracker.utils.ActionBarHelper

class DonateActivity extends SActivity {

    override def onCreate(bundle:Bundle) {
        super.onCreate(bundle)

        this.setContentView(R.layout.activity_donate)

        ActionBarHelper.colorActionBar(this)
        ActionBarHelper.enableHomeAsUp(this)

        val webView = this.findViewById(R.id.webview).asInstanceOf[WebView]
        webView.loadUrl(this.getString(R.string.donate_url))
    }

    override def onOptionsItemSelected(item:MenuItem):Boolean = {
        item.getItemId() match {
            case android.R.id.home => {
                finish();

                return true
            }
        }

        return super.onOptionsItemSelected(item)
    }
}